src/components/grid-aware/ImageHeader/ImageHeader.module.css

```diff
@@ -110,6 +127,11 @@
     margin-bottom: 20px;
     margin-right: 0;
   }
+
+  .descriptionWrapper {
+    font: var(--font-body-medium);
```

Comment:
This is the same font size used in the desktop view. Recall that we have the font variables themselves change based on the screen size, so this will automatically be adjusted for smaller screen sizes. Could you remove this line?